Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM --platform=linux/amd64 ubuntu:20.04
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential
+
+ADD . /repo
+WORKDIR /repo
+RUN ./build.sh
```

Comment:
Is it possible to separate this into a build and package stage? (I don't know how many essential libraries are compiled during `build.sh`)